roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -81,12 +81,6 @@ netchecker_etcd_memory_limit: 256M
 netchecker_etcd_cpu_requests: 100m
 netchecker_etcd_memory_requests: 128M
 
-# SecurityContext when PodSecurityPolicy is enabled
-netchecker_agent_user: 1000
-netchecker_server_user: 1000
-netchecker_agent_group: 1000
-netchecker_server_group: 1000
-
```

Comment:
Does keeping `netchecker_agent_user` is better? Changing to 
```
#netchecker_agent_user: 1000
#netchecker_server_user: 1000
#netchecker_agent_group: 1000
#netchecker_server_group: 1000
```

Reply:
Hum, either we keep it or we scrape it (and yeah we can keep it, it does not affect the rest, I only removed it initially because it looked to me this was only for PSP given the comments)

But I'd rather avoid keeping commented out stuff anywhere :laughing:

Reply:
I wish that we can remove netchecker completely. As it's out-of-date and maybe insecure.

Reply:
That's what we use in CI though ? I've no idea what the replacement would be.
Anyway, that would be another cleanup PR :+1: